### DISCUSSION TITLE
Upgraded to 2.10.2 and reports not working

### DISCUSSION THREAD

**Discussion by jadams3710** [netbox-community/netbox#5562] [Q&A] [answered]
I recently upgraded to 2.10.2 and now none of the reports are not working. 
Here is a screenshot of my error. Not sure where to start. Any help would be appreciate. Not urgent but would be nice to have reports back.
![netbox_error](https://user-images.githubusercontent.com/55246184/103423436-21c45c00-4b6c-11eb-880c-d1c5786a5efb.jpg)

**Comment by jadams3710** [comment, marked as answer]
Ended up commenting out - CustomFieldValue from circuit_audits.py and reports are working now.

**Comment by candlerb** [comment]
The implementation of Custom Fields changed in Netbox 2.10 - this is mentioned in the [release notes](https://github.com/netbox-community/netbox/releases/tag/v2.10.0).  Any code in reports or scripts which uses them will need to be changed.

**Reply by candlerb** [reply]
I have pushed a fixed version of this report to https://github.com/netbox-community/reports/pull/45
